scm-ui/ui-components/src/layout/Page.tsx

```diff
@@ -61,6 +61,7 @@ const MarginLeft = styled.div`
 const FlexContainer = styled.div`
   display: flex;
   flex-direction: row;
+  height: 2.25rem;
```

Comment:
Actually i mean default height. The repositoryFormSwitcher gets hard to handle with responsive css and this would be the easiest fix for this. This css only effects the "title bar". I already checked every other page and it seems fine.